src/class-fields/computed-name-empty-var-empty-function.case

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2016 the V8 project authors, 2017 Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: Computed name var not initialized, empty function
+info: |
+  1.1 New Productions
+
+  [...]
+
+  FieldDefinitionList [Yield, Await]:
+    FieldDefinition [?Yield, ?Await]
+    FieldDefinitionList  [?Yield, ?Await], FieldDefinition [?Yield, ?Await]
+template: default
+includes: [propertyHelper.js]
+---*/
+
+//- body
+  ['a']; b(){}
+
+//- testfield
+  isEqualTo(this, "a", undefined);
+  verifyEnumerable(this, "a");
+  verifyWritable(this, "a");
+  verifyConfigurable(this, "a");
+
+  isEqualTo(this, "b", undefined);
```

Comment:
it would be nice to have before the body:

```js
let c = 'a';
```

and then check the computed property name based on the value of c:

```js
//- body
  [c]; b(){}
```